README.md

```diff
@@ -1,79 +1,196 @@
+- Opt-in to [Tusky Beta on Google Play](https://play.google.com/store/apps/details?id=com.keylesspalace.tusky)
 
-**A:** You can sign up for the [Google Play beta, via this link](https://play.google.com/store/apps/details?id=com.keylesspalace.tusky)
-##
+Nightly releases are available on Google Play and F-Droid:
 
-**Q:** What about Nightly releases?
+- Install [Tusky Nightly from Google Play](https://play.google.com/store/apps/details?id=com.keylesspalace.tusky.test)
+- In the F-Droid settings, select My Apps > Repositories, and add a new repository with the link `https://releases.nailyk.fr/repo/`
 
-**A:** The Tusky nightly is [available on Google Play](https://play.google.com/store/apps/details?id=com.keylesspalace.tusky.test) as well. On F-Droid, [add this repo](https://releases.nailyk.fr/repo/).
-##
+### Why is the Tusky release on F-Droid out of date?
 
-**Q:** The likes and boosts aren't displaying the same numbers as on the web?
+F-Droid builds Tusky from the source code when a new release is made available. This can take longer than it takes for the new release to available on Google Play.
 
-**A:** This is not a Tusky bug, it's because boosts and likes do not federate perfectly. I.e., if you and the user whom you see this on, are not on the same server you will see different numbers on Tusky and on their message on the web.
-##
+## Managing my account
 
-**Q:** How do I remove an account from Tusky?
+### Can I create an account with Tusky?
 
-**A:** Just select the account, then log out.
-##
+Not yet, you must create an account via a service like https://joinmastodon.org/ first.
 
-**Q:** My instance has a custom character limit, but Tusky allows only 500 characters
+Once you have done this you can then log in to the account in Tusky.
 
-**A:** Your instance needs to make this information public via the `max_toot_chars` property of the `/api/v1/instance` endpoint. Contact your admin to change this. See e.g. here https://pleroma.site/api/v1/instance for a config that works with Tusky.
-##
+### Can I remove my account with Tusky?
 
-**Q:** Will you block more domains than the Gab related ones?
+You can sign out of your account with Tusky. But you will need to follow your instance's procedure to fully delete your account.
 
-**A:** The size of Gab is pretty unique and we have no time to research smaller instances, so most likely no, but it's always a possibility.
-##
+### I'm trying to log in to my account, and am being rick-rolled. Why?
 
-**Q:** Is this censorship/against free speech etc.? 
+Tusky does not support usage by accounts on some [servers](https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/res/values/donottranslate.xml#L160) because of their association with hate groups.
 
-**A:** No because of three reasons, 1, hate against marginalised groups is not an opinion; 2, Gab can still be accessed by other means and Tusky can be forked; 3, Tusky is no government, so it cannot take away your basic rights.
-##
```

Comment:
Because it wasn't written from the perspective of the user.

And historically, I've found that the more specific the justifications become, the more abusive users will try and weasel around them.